frontends/web/src/routes/account/send/send.tsx

```diff
@@ -494,7 +500,6 @@ class Send extends Component<Props, State> {
                 coinCode={account.coinCode}
                 transactionDetails={waitDialogTransactionDetails}
               />
-              <SendResult result={sendResult} />
```

Comment:
I am not sure if we should do this commit, it changes the behavior as it prevents the other components from being rendered, especially FeeTargets or CoinContorl UTXOs component.

This could lead to problems when there was an error and the user goes back, the selected UTXO's in the subcomponent might not be selected anymore

I think it is cleaner this way, but it is not requierd.. need to do more testing.

Reply:
As CoinControl is currently not a "controlled" component it would loose the state and selected UTXOs wouldn't show again as checked if the user had an error and goes back ith "Edit transaction".

I'll take a note to make CoinControl a controlled component and remove from this PR, cc @shonsirsha